src/main/resources/changelog/changes/v3.3/create-reindex-entity-tables.xml

```diff
@@ -0,0 +1,282 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.27.xsd">
+
+  <changeSet id="MSEARCH-793@@create-instances-table" author="Mukhiddin_Yusupov">
+    <preConditions>
+      <not>
+        <tableExists tableName="instances"/>
+      </not>
+    </preConditions>
+
+    <comment>Create instances table</comment>
+
+    <createTable tableName="instances">
+      <column name="instance_id" type="VARCHAR(255)">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="pk_instances"/>
+      </column>
+      <column name="tenant_id" type="VARCHAR(255)">
+        <constraints nullable="false"/>
+      </column>
+      <column name="shared" type="BOOLEAN">
+        <constraints nullable="false"/>
+      </column>
+      <column name="is_bound_with" type="BOOLEAN">
+        <constraints nullable="false"/>
+      </column>
+      <column name="instance_json" type="jsonb">
+        <constraints nullable="false"/>
+      </column>
+    </createTable>
+  </changeSet>
+
+  <changeSet id="MSEARCH-793@@create-holdings-table" author="Mukhiddin_Yusupov">
+    <preConditions>
+      <not>
+        <tableExists tableName="holdings"/>
+      </not>
+    </preConditions>
+
+    <comment>Create holdings table</comment>
+
+    <createTable tableName="holdings">
+      <column name="holding_id" type="VARCHAR(255)">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="pk_holdings"/>
+      </column>
+      <column name="tenant_id" type="VARCHAR(255)">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="pk_holdings"/>
+      </column>
+      <column name="instance_id" type="VARCHAR(255)">
+        <constraints nullable="false"/>
+      </column>
+      <column name="holding_json" type="jsonb">
+        <constraints nullable="false"/>
+      </column>
+    </createTable>
+  </changeSet>
+
+  <changeSet id="MSEARCH-793@@create-idx_holdings_instance_id" author="Mukhiddin_Yusupov">
```

Comment:
Let's move creating indexes into creating table changes to make it one transaction.